crates/networking/p2p/rlpx/eth.rs

```diff
@@ -109,3 +109,314 @@ impl RLPxMessage for StatusMessage {
         })
     }
 }
+
+// https://github.com/ethereum/devp2p/blob/master/caps/eth.md#getblockbodies-0x05
+#[derive(Debug)]
+pub(crate) struct GetBlockBodies {
+    // id is a u64 chosen by the requesting peer, the responding peer must mirror the value for the response
+    // https://github.com/ethereum/devp2p/blob/master/caps/eth.md#protocol-messages
+    id: u64,
+    block_hashes: Vec<BlockHash>,
+}
+
+impl GetBlockBodies {
+    pub fn build_from(id: u64, block_hashes: Vec<BlockHash>) -> Self {
```

Comment:
Why not `new`?

Reply:
Ah, thought it was the convention for this type of structs, followed the one of [StatusMessage](https://github.com/lambdaclass/ethereum_rust/pull/710/files#diff-03afa5c356f9f73cab4212954b4597f70985b12c43856752c2af11caeac9ecf3R29), changing them to `new`